pallets/parachain-staking/src/test_reward_payout.rs

```diff
@@ -146,9 +150,182 @@ fn end_to_end_happy_path() {
 
             // check that distributing rewards clears awarded points
             assert!(
-                ParachainStaking::awarded_pts(era_blocks_have_been_authored, collator1).is_zero()
+                ParachainStaking::awarded_pts(ERA_BLOCKS_HAVE_BEEN_AUTHORED, COLLATOR1).is_zero()
             );
         });
 }
 
-// TODO: add failing tests for payout logic (next PR)
+// This function will setup the payments so both collators get the same reward
+fn set_reward_pot_and_trigger_payout(block_author_era: u32, destiantion_era: u64) -> (u128, u128) {
+    pay_gas_for_transaction(&TX_SENDER, TIP);
+
+    set_author(block_author_era, COLLATOR1, COLLATOR1_POINTS);
+    set_author(block_author_era, COLLATOR2, COLLATOR2_POINTS);
+
+    roll_to_era_begin(destiantion_era);
```

Comment:
```suggestion
    roll_to_era_begin(destination_era);
```
And the corresponding changes